test/unit/model/virtualbox-test.js

```diff
@@ -288,34 +287,21 @@ describe('Virtualbox installer', function() {
     });
 
     it('should add option \'detected\' with detected version and location', function() {
-      return new Promise((resolve, rejects)=> {
-        installer.detectExistingInstall(resolve);
-      }).then(()=>{
+      installer.detectExistingInstall().then(()=> {
```

Comment:
return statement is required here. It should return promise or mocha would not wait for anything,